easybuild/easyconfigs/c/CrayCCE/CrayCCE-5.1.29.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'Toolchain'
+
+name = 'CrayCCE'
+version = '5.1.29'
+
+homepage = 'http://hpcugent.github.io/easybuild/'
```

Comment:
make this `(none)`, like we do for the other toolchains?

or maybe `http://http://docs.cray.com/`?